wdtk-rdf/src/main/java/org/wikidata/wdtk/rdf/RdfConverter.java

```diff
@@ -363,6 +364,21 @@ void writeTermTriples(Resource subject, URI predicate,
 		}
 	}
 
+	void writeStatementRankTriple(Resource subject, StatementRank rank) {
+		String rankRepresentation = rankToRdfRepresentation(rank);
+		if (rankRepresentation != null){
+			try {
+				this.rdfWriter.writeTripleStringObject(subject, RdfWriter.WB_RANK,
+						rankRepresentation);
```

Comment:
Ranks should be represented by resources (URIs) rather than by strings. The URIs should use local names PreferredRank, NormalRank, DeprecatedRank and be defined in the vocabulary class.